doc/connectivity-semantics-and-api.md

```diff
@@ -75,6 +90,15 @@ corresponding reasons. Empty cells denote disallowed transitions.
     <td></td>
     <td></td>
     <td></td>
+    <td>Shutdown triggered by application.</td>
+  </tr>
+  <tr>
+    <th>IDLE</th>
+    <td>Any new RPC activity on the channel</td>
```

Comment:
"Any new RPCs on the channel"? If we receive a GOAWAY existing RPCs can continue, but we would still be in IDLE.